src/components/swap/AdvancedSwapDetails.test.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+  TEST_ALLOWED_SLIPPAGE,
+  TEST_TOKEN_1,
+  TEST_TRADE_EXACT_INPUT,
+  TEST_TRADE_EXACT_OUTPUT,
+  toCurrencyAmount,
+} from 'test-utils/constants'
+import { fireEvent, render, screen } from 'test-utils/render'
+
+import { AdvancedSwapDetails } from './AdvancedSwapDetails'
+
+jest.mock('@web3-react/core', () => {
+  const web3React = jest.requireActual('@web3-react/core')
+  return {
+    ...web3React,
+    useWeb3React: () => ({
+      chainId: 1,
+    }),
+  }
+})
+
+describe('AdvancedSwapDetails.tsx', () => {
+  it('matches base snapshot', () => {
+    const { asFragment } = render(
+      <AdvancedSwapDetails trade={TEST_TRADE_EXACT_INPUT} allowedSlippage={TEST_ALLOWED_SLIPPAGE} />
+    )
+    expect(asFragment()).toMatchSnapshot()
+  })
+
+  it('test trade with exact input', async () => {
```

Comment:
`it` descriptors should describe what the test does, so in this case I think you'd want `it('should render the correct copy on mouseover')`

Reply:
done